### PR TITLE
Added Eslint

### PR DESCRIPTION
**Pull request by magsout**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,18 +22,16 @@
...
+    "postcss": "^4.1.16"
   },
   "devDependencies": {
     "css-whitespace": "^1.1.0",
-    "jscs": "^1.6.2",
-    "jshint": "^2.5.6",
-    "tape": "^3.0.0"
+    "tape": "^4.0.1",
+    "eslint": "^1.0.0-rc-1",
+    "eslint-friendly-formatter": "^1.1.0"
```
Comment (MoOx) [inline package.json]: the command "npm run lint" defined don't use this formatter, I think for small projects like this we don't need it.

**Comment by MoOx** [comment]
Except my small comment, it looks good!

**Comment by magsout** [comment]
@MoOx oh thanks, good to know ;) 
remove and update thanks for the review

**Comment by MoOx** [comment]
👌🏼

**Pull request merged**